Sources/FoundationEssentials/Data/Data.swift

```diff
@@ -30,10 +30,25 @@ private func malloc_good_size(_ size: Int) -> Int {
     return size
 }
 
+#elseif canImport(ucrt)
+import ucrt
+
+private func malloc_good_size(_ size: Int) -> Int {
+    return size
+}
+
+#endif
+
+#if os(Windows)
```

Comment:
(nit) should this be `#elseif canImport(WinSDK)...#endif` to be consistent?

Reply:
I suppose we can do that.  Honestly, I think that `canImport` is generally the wrong thing to use for identifying the platform.

Reply:
Let's follow up on this subsequently.  I think that we will have a few more of these, and we can homogenise them properly in a single pass.